src/components/number/number.jsx

```diff
@@ -55,12 +55,19 @@ function Number({
   const minimumFractionDigits = getFractionDigits(tickDecimals, valueMinDecimals, valueDecimals);
   const maximumFractionDigits = getFractionDigits(tickDecimals, valueMaxDecimals, valueDecimals);
   const formattedNumber = formatNumber(value, { minimumFractionDigits, maximumFractionDigits });
+  const absFormattedNumber = formatNumber(Math.abs(value), { minimumFractionDigits, maximumFractionDigits });
+  const ariaSign = (value < 0) ? '−' : '';
```

Comment:
I changed to this character instead of trying to "build" it with "&minus;", because I do not get that way to work for the aria-label. @bstream